src/MoBi.UI/Views/EditSpatialStructureView.cs

```diff
@@ -62,6 +62,9 @@ private void tabSelectionChanging(TabPageChangingEventArgs e)
 
       public void SetEditView(IView view)
       {
+         if (view == null && splitHierarchyEdit.Panel2.Controls.Count > 0)
+            splitHierarchyEdit.Panel2.Controls.RemoveAt(0);
```

Comment:
What about changing this to remove all child controls. I know that by convention we always use this `FillWith` which places only one control, but just in case this is not always the case we want to empty this panel.